simpleocr/files.py

```diff
@@ -28,6 +29,18 @@ def try_extensions(path, extensions):
     return None
 
 
+def find_file(file_name):
```

Comment:
Isn't this related to this PR because it is directly caused by the PR? I did not have any trouble opening custom files before I made the packaging changes... Still, I will remove the commit from this branch then, so we can work it out later.